### ISSUE TITLE
Add a captcha to guest comments

### ISSUE THREAD

**Issue by hemebond** [hemebond/quaddicted#22]
Maybe https://pypi.org/project/django-simple-captcha/

**Comment by SpiritQuaddicted** [comment]
Is there something that uses plain text questions? We've had great success with the current 3-4 Quake-specific questions. It gives a nice vibe :)

**Comment by hemebond** [comment]
@SpiritQuaddicted Do you have any data on how many bots have been stopped by the questions?

**Comment by SpiritQuaddicted** [comment]
No, we don't track much. There have been very few spammers coming through though.

**Comment by hemebond** [comment]
Others in Discord have expressed the desire to only allow logged-in users to leave comments; I share that sentiment as well. Makes other systems much simpler (e.g., highlighting possible impersonations).

**Comment by SpiritQuaddicted** [comment]
We have had no issues on Quaddicted and I am a strong believer in privacy. Participating should not require an account.
